### ISSUE TITLE
Small Bug with Fan Lights that have dimming...

### ISSUE THREAD

**Issue by methnen** [aarons22/homebridge-bond#129]
**Describe the bug**
Bug is super easy to reproduce:

1. Turn on light from HomeKit
2. Dim light until its off from HomeKit
3. Try to turn light back on by ramping the dimming back up via HomeKit

The light does not come back on and I start getting "No Response" warning inside of HomeKit.

I can get it working again if I go back to the tile screen of HomeKit and toggle the light on and off a few times.

My wild guess is that somewhere on the Homebridge end it's not sending a turn light back on signal to the Bond when going from off to some ramped up/dimmed value. But HomeKit then thinks the light is on and gets confused by whatever it's getting for status from Homebridge.

I'm including a log capture of me turning the light on, dimming it to zero and then attempting to ramp it back up.

 - Plugin version v3.2.3
 - Bond firmware version v2.15 (Bond for Home)
 - How homebridge is run: Homebridge UI
 
**Logs**
```
[1/7/2021, 3:56:28 PM] [Bond] Request (185db7d1edf9c000) [put http://192.168.1.30/v2/devices/1/actions/ToggleLight]
[1/7/2021, 3:56:28 PM] [Bond] UDP Message received from 192.168.1.30:30007 - {"B":"KSMJWDC27015","t":"devices/1/state","i":"050000b0d9f70b36","f":100,"s":200,"m":0,"x":"bond","b":{"power":0,"speed":6,"light":1,"brightness":84,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"fa2a6228"}}
[1/7/2021, 3:56:28 PM] [Bond] [Artemis Fan] Received new state: {"power":0,"speed":6,"light":1,"brightness":84,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"fa2a6228"}
[1/7/2021, 3:56:28 PM] [Bond] Response (185db7d1edf9c000) [put http://192.168.1.30/v2/devices/1/actions/ToggleLight] - {"_":"ba1275ad"}
[1/7/2021, 3:56:28 PM] [Bond] [Artemis Fan] Set light power: true
[1/7/2021, 3:56:29 PM] [Bond] UDP Message received from 192.168.1.30:30007 - {"B":"KSMJWDC27015","t":"devices/1/state","i":"050000605eb3e594","f":100,"s":200,"m":0,"x":"bond","b":{"power":0,"speed":6,"light":1,"brightness":84,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"ef913ef8"}}
[1/7/2021, 3:56:29 PM] [Bond] [Artemis Fan] Received new state: {"power":0,"speed":6,"light":1,"brightness":84,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"ef913ef8"}
[1/7/2021, 3:56:32 PM] [Bond] Request (185db7d1f1e6c000) [put http://192.168.1.30/v2/devices/1/actions/SetBrightness] - body: {"argument":39}
[1/7/2021, 3:56:32 PM] [Bond] UDP Message received from 192.168.1.30:30007 - {"B":"KSMJWDC27015","t":"devices/1/state","i":"050000471a53b701","f":100,"s":200,"m":0,"x":"bond","b":{"power":0,"speed":6,"light":1,"brightness":39,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"1a331379"}}
[1/7/2021, 3:56:32 PM] [Bond] [Artemis Fan] Received new state: {"power":0,"speed":6,"light":1,"brightness":39,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"1a331379"}
[1/7/2021, 3:56:32 PM] [Bond] Response (185db7d1f1e6c000) [put http://192.168.1.30/v2/devices/1/actions/SetBrightness] - {"argument":39,"_":"40d47c86"}
[1/7/2021, 3:56:32 PM] [Bond] [Artemis Fan] Set light brightness: 39
[1/7/2021, 3:56:32 PM] [Bond] Request (185db7d1f2284000) [put http://192.168.1.30/v2/devices/1/actions/SetBrightness] - body: {"argument":16}
[1/7/2021, 3:56:32 PM] [Bond] UDP Message received from 192.168.1.30:30007 - {"B":"KSMJWDC27015","t":"devices/1/state","i":"05000098cedd919b","f":100,"s":200,"m":0,"x":"bond","b":{"power":0,"speed":6,"light":1,"brightness":16,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"2795f94b"}}
[1/7/2021, 3:56:32 PM] [Bond] [Artemis Fan] Received new state: {"power":0,"speed":6,"light":1,"brightness":16,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"2795f94b"}
[1/7/2021, 3:56:32 PM] [Bond] Response (185db7d1f2284000) [put http://192.168.1.30/v2/devices/1/actions/SetBrightness] - {"argument":16,"_":"40d47c86"}
[1/7/2021, 3:56:32 PM] [Bond] [Artemis Fan] Set light brightness: 16
[1/7/2021, 3:56:32 PM] [Bond] Request (185db7d1f26a0000) [put http://192.168.1.30/v2/devices/1/actions/ToggleLight]
[1/7/2021, 3:56:32 PM] [Bond] UDP Message received from 192.168.1.30:30007 - {"B":"KSMJWDC27015","t":"devices/1/state","i":"05000079ee899605","f":100,"s":200,"m":0,"x":"bond","b":{"power":0,"speed":6,"light":0,"brightness":16,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"dd1f16ed"}}
[1/7/2021, 3:56:32 PM] [Bond] [Artemis Fan] Received new state: {"power":0,"speed":6,"light":0,"brightness":16,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"dd1f16ed"}
[1/7/2021, 3:56:32 PM] [Bond] Response (185db7d1f26a0000) [put http://192.168.1.30/v2/devices/1/actions/ToggleLight] - {"_":"ba1275ad"}
[1/7/2021, 3:56:32 PM] [Bond] [Artemis Fan] Set light power: false
[1/7/2021, 3:56:33 PM] [Bond] UDP Message received from 192.168.1.30:30007 - {"B":"KSMJWDC27015","t":"devices/1/state","i":"0500007c046218d3","f":100,"s":200,"m":0,"x":"bond","b":{"power":0,"speed":6,"light":0,"brightness":16,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"3874d5ff"}}
[1/7/2021, 3:56:33 PM] [Bond] [Artemis Fan] Received new state: {"power":0,"speed":6,"light":0,"brightness":16,"brightness_cycle_phase":-1,"timer":0,"breeze":[0,50,50],"direction":1,"_":"3874d5ff"}

**Comment by methnen** [comment]
You're gonna hate me, but after updating I'm not seeing any of my Bond devices showing up anymore in Homebridge or Homekit.

I double checked my Bond settings in Homebridge and they're all still correct so I'm not sure what's going on.

I'm seeing nothing in the log when I turn the fan light on/off in from the Bond app either so it's like it just lost it's connection entirely to Homebridge.

Plugin version v3.2.4
Bond firmware version v2.15 (Bond for Home)
How homebridge is run: Homebridge UI
Homebridge version v1.3.4
Node.js version v14.16.0
Npm version v6.14.11

**Comment by methnen** [comment]
Tried rebooting a couple of times and that doesn't seem to clear things up either.

**Comment by aarons22** [comment]
yeahhh sorry about that. I borked 3.2.4. I just pushed up a fix in 3.2.5!

**Comment by methnen** [comment]
It works! And yes the bug seems to fixed as well.
